code/game/machinery/hologram.dm

```diff
@@ -56,9 +56,10 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 
 	var/allow_ai = TRUE
 
-/obj/machinery/hologram/holopad/Initialize()
-	. = ..()
-	desc = "It's a floor-mounted device for projecting holographic images. Its ID is '[loc.loc]'"
+/obj/machinery/hologram/holopad/New()
+	..()
+	if(loc)
+		desc = "It's a floor-mounted device for projecting holographic images. Its ID is '[loc.loc]'"
 
```

Comment:
Might want to revert to the Initialize version and check if this change is needed

Reply:
Reverting this is fine, it should be fixed with this in init